src/components/ElectionMap.js

```diff
@@ -253,13 +241,11 @@ class ElectionMap extends SvgChart {
       .attr("y", -rectSide / 2)
       .attr("width", rectSide)
       .attr("height", rectSide)
-      .style("transition", "all .2s ease-out")
```

Comment:
this is only css

Reply:
Based on profile above, the most severe bottleneck is browser painting, not stuck on JavaScript execution krub.